src/Uno.UI.RemoteControl.VS/Commands/UnoMenuCommand.cs

```diff
@@ -124,10 +136,21 @@ private void OnBeforeQueryStatusDynamicItem(object sender, EventArgs args)
 		matchedCommand.MatchedCommandId = 0;
 	}
 
-	private static int GetCurrentPosition(DynamicItemMenuCommand matchedCommand) =>
-			// The position of the command is the command ID minus the ID of the root dynamic start item.
-			matchedCommand.MatchedCommandId == 0 ? 0 : matchedCommand.MatchedCommandId - (int)DynamicMenuCommandId;
+	private static int GetCurrentPosition(DynamicItemMenuCommand matchedCommand)
+		// The position of the command is the command ID minus the ID of the root dynamic start item.
+		=> matchedCommand.MatchedCommandId == 0 ? 0 : matchedCommand.MatchedCommandId - (int)DynamicMenuCommandId;
 
 	private bool TryGetCommandRequestIdeMessage(DynamicItemMenuCommand matchedCommand, [NotNullWhen(true)] out AddMenuItemRequestIdeMessage result)
 		=> (result = CommandList.Skip(GetCurrentPosition(matchedCommand)).FirstOrDefault()) != null;
+
+	public void Dispose()
+	{
+		CommandService.RemoveCommand(_dynamicMenuCommand);
```

Comment:
Is it ok passing null into here? (could not find `RemoveCommand` implementation to validate it myself

Reply:
Fair! Those apis are not nullable enable, it does not show in compiler.